Form-Controls/styles.css

```diff
@@ -0,0 +1,115 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+header {
+  display: flex;
+}
+
+body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  background: url(https://ae01.alicdn.com/kf/HTB1YjcPat.LL1JjSZFEq6AVmXXaO/TOP-QUALITY-Summer-Solid-Blank-T-Shirt-Short-Sleeve-Cotton-T-shirt-Women-2017-Fashion-Tshirt.jpg);
+  background-size: cover;
+}
+
+.container {
+  width: 100%;
+  max-width: 800px;
+  background: rgba(114, 80, 80, 0.5);
+  padding: 28px;
+  margin: 0 28;
+  border-radius: 10px;
+  box-shadow: inset -2px 2px 2px white;
+}
+
+.form-title {
+  font-size: 26px;
+  font-weight: 600;
+  text-align: center;
+  padding-bottom: 6px;
+  color: white;
+  text-shadow: 2px 2px 2px black;
+  border-bottom: solid 1px white;
+}
+
+.main-costumer-details {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 20px 0;
+}
+
+.customer-input-box:nth-child(2n) {
+  justify-content: end;
+}
+
+.customer-input-box {
+  display: flex;
+  flex-wrap: wrap;
+  width: 50%;
+  padding-bottom: 15px;
+}
+
+.customer-input-box label {
+  width: 95%;
+  color: white;
+  font-size: 20px;
+  font-weight: 400;
+  margin: 5px 0;
+}
+
+.customer-input-box input {
+  height: 40px;
+  width: 95%;
+  border-radius: 7px;
+  outline: none;
+  border: 1px solid grey;
+  padding: 0 10px;
+}
+
+.color-title {
+  color: white;
+  font-size: 24px;
+  font-weight: 600;
+  border-bottom: 1px solid white;
+}
+
+.color-category {
+  margin: 15px 0;
+  color: white;
+}
+
+.color-category label .color-category input {
+  cursor: pointer;
+}
+
+@media (max-width: 600px) {
+  .container {
+    min-width: 280px;
+  }
+  .customer-input-box {
+    margin-bottom: 12px;
+    width: 100%;
+  }
+  .customer-input-box:nth-child(2n) {
+    justify-content: space-between;
+  }
+  .color-category {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+  .main-costumer-details {
```

Comment:
'costomer' is spelt different to the element's class in the HTML, so I'm not sure if this CSS is being used by the page